### DISCUSSION TITLE
Powered by Airbyte: White label

### DISCUSSION THREAD

**Discussion by w14** [airbytehq/airbyte#32474] [Connector Questions] [answered]
Is there a way to add airbyte to my product so users can set up their own connectors, like "Powered by Fivetran"?

**Comment by rileybrook** [comment, marked as answer]
Hey @w14 👋 

Please check out [Powered by Airbyte](https://airbyte.com/solutions/powered-by-airbyte)! If you have any questions about it, feel free to reach out to me at riley@airbyte.io 🙂

**Reply by w14** [reply]
1. Is pricing affected by manual triggers?
3. Does Powered by Airbyte support private custom connectors (https://docs.airbyte.com/operator-guides/using-custom-connectors)?

These docs are great https://reference.airbyte.com/reference/powered-by-airbyte. They even have little animations.
